events/importer/kulke.py

```diff
@@ -184,7 +184,7 @@ def parse_age_range(secondary_headline):
     if not isinstance(secondary_headline, str):
         return (None, None)
 
-    pattern = r'^\D*(\d{1,2}).(\d{1,2}).[v.|år].+$'
+    pattern = r'^\D*(\d{1,2}).(\d{1,2}).(v|år).*$'
```

Comment:
I'm still a little bit worried about the "." between numbers as it matches numbers.  Maybe "[[:punct:]]"?  Anyway, this is a minor detail that does not warrant action IMO.